aea/crypto/ethereum.py

```diff
@@ -238,3 +238,15 @@ def is_transaction_settled(self, tx_digest: str) -> bool:
         if tx_status is not None:
             is_successful = True
         return is_successful
+
+    def validate_transaction(self, tx_digest: str, random_message: str) -> bool:
```

Comment:
This should be part of the ledger api class in my opinion as it is related to the ledger interaction, not the crypto/account management